frontend/src/components/LinkInformations/RedeemLoggedOut.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="redeem-logged-out">
+    <b-jumbotron bg-variant="muted" text-variant="dark" border-variant="info">
+      <h1>
+        {{ user.firstName }}
+        {{ $t('transaction-link.send_you') }} {{ amount | GDD }}
+      </h1>
+      <b>{{ memo }}</b>
+    </b-jumbotron>
```

Comment:
I think this block would be suited to be a component, since it is the same in the three files: Redeem*.vue